library/src/main/java/com/ibm/research/drl/dpt/anonymization/constraints/KAnonymity.java

```diff
@@ -62,4 +62,9 @@ public void sanityChecks(IPVDataset originalDataset) {
     public void initialize(IPVDataset dataset, List<ColumnInformation> columnInformationList) {
         sanityChecks(dataset);
     }
+
```

Comment:
same here, should upgrade copyright to 2022?

Reply:
No real changes to this file